test/primer/classify_test.rb

```diff
@@ -341,7 +341,7 @@ def refute_generated_class(input)
     assert_nil(Primer::Classify.call(**input)[:class])
   end
 
-  def assert_allocations(count, within:, &block)
```

Comment:
We don't need this argument? I would have thought we did. cc @BlakeWilliams

Reply:
I think we don't need it since we use `yield`

Reply:
I forgot where I picked it up, but sometimes `&block` is added to method arguments to signal that they accept a block without requiring API consumers to look at the source.

I'm not opposed to removing it though, if that's what our linter recommends.